### PR TITLE
Add/parse configuration

### PR DESCRIPTION
**Pull request by haleyharelson**

### CODE REVIEW

File: app/src/main/java/com/fbu/fbuteam/ParseApp.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import com.parse.Parse;
+import com.parse.ParseObject;
+
+public class ParseApp extends Application {
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        //sets up Parse server
```
Comment (LeoNero) [inline app/src/main/java/com/fbu/fbuteam/ParseApp.java]: remove this comment

File: app/src/main/java/com/fbu/fbuteam/ParseApp.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        final Parse.Configuration configuration = new Parse.Configuration.Builder(this)
```
Comment (LeoNero) [inline app/src/main/java/com/fbu/fbuteam/ParseApp.java:17]: You can put all this part in a function called "setupParse"

File: app/src/main/res/layout/activity_signup.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/email_et"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/prompt_email_signup"
+            android:inputType="text" />
+    </com.google.android.material.textfield.TextInputLayout>
+
```
Comment (LeoNero) [inline app/src/main/res/layout/activity_signup.xml:29]: Maybe use only one empty line instead of two?

**Review by ghost — COMMENTED** [review]
Spacing, gradle.properties lines 9-10 .

File: app/src/main/java/com/fbu/fbuteam/SignUpActivity.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
+package com.fbu.fbuteam;
```
Comment (LeoNero) [inline app/src/main/java/com/fbu/fbuteam/SignUpActivity.java]: I would suggest rewriting it as: https://gist.github.com/LeoNero/9bb5fb07d8f544a44c118edbef3e3921

**Review by LeoNero — CHANGES_REQUESTED** [review]

**Review by LeoNero — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**